src/components/common/RouterLinkNetwork.vue

```diff
@@ -36,9 +36,7 @@ export default {
     ...mapGetters([`network`]),
     url() {
       if (this.network.length) {
-        const networkName = this.network.includes(`mainnet`)
-          ? this.network.slice(0, -8)
-          : this.network
+        const networkName = this.network.replace(/-mainnet$/, ``)
```

Comment:
Cool

Reply:
this forces us to have the same id schema for every network. :/

Reply:
I think we should avoid string manipulations on IDs wherever possible

Reply:
> I think we should avoid string manipulations on IDs wherever possible

Why?

Reply:
because IDs are IDs and this forces us to a naming schema. IDs should not need to follow a naming schema.

Reply:
But since then, how we are going to remove useless parts like mainnet?

Reply:
probably doing a lookup

Reply:
I am not sure how to do this. 

If the ID doesn't follow any naming schema/pattern, then how can we know which is the lexeme of the network and which the "useless" parts.

Maybe I didn't understand you correctly, because I don't see any possible solution to this

Reply:
there should not be "parts" in an ID. it could be random. that is why I forced everyone to use the `testnet` variable from the networks response. This is the same case. If we want the URL to be different then the ID, we should do a dictionary lookup. Or we just pass the `route` from the API. which is preferable as we don't want network specific logic on the client.

Reply:
Maybe we can just store path in our db?

Reply:
Updated

Reply:
#slugs